akka-docs/rst/java/lambda-persistence.rst

```diff
@@ -932,6 +936,37 @@ directory. This can be changed by configuration where the specified path can be
 Note that it is not mandatory to specify a snapshot store plugin. If you don't use snapshots
 you don't have to configure it.
 
+.. _journal-proxy:
+
+Journal Proxy
+-------------
+
+A journal proxy allows sharing of journals and snapshot stores across multiple actor systems (on the same or on
+different nodes). This, for example, allows persistent actors to failover to a backup node and continue using the
+shared journal instance from the backup node. The journal proxy works by forwarding all the journal/snapshot store
+messages to a single, shared, persistence plugin instance, and therefor supports any use case supported by the proxied
+plugin (for example, :ref:`Persistence Query <persistence-query-scala>`).
```

Comment:
ah, that's just because the leveldb query is using the write journal to serve the queries, I don't think that will be a normal solution, so we should not promise anything about that in the docs